conf/chunkserver.conf

```diff
@@ -158,6 +157,8 @@ storeng.sync_write=false
 #
 # QoS settings
 #
+# 限制inflight io数量，一般是5000
+copyset.max_inflight_requests=5000
```

Comment:
the prefix `copyset` means this configuration belongs to the copyset module, but original issue #1255 said it's not.

so, please modify the prefix, like `chunkserver` or something else, rather than the location.

Reply:
ok i have corrected the error